### DISCUSSION TITLE
`YamlDecoder[Duration]` ?

### DISCUSSION THREAD

**Discussion by ramytanios** [VirtusLab/scala-yaml#270] [Q&A]
Hello, 

I am considering switching from `pureconfig` to `yaml` for my config files. However `pureconfig` supports decoding `Duration` objects. Can we have that too for `yaml`?

**Comment by tgodzik** [comment]
You can probably implement YamlCodec for that, but I don't think we have it out of the box.

**Reply by ramytanios** [reply]
Do you plan to implement this? Thanks

**Reply by tgodzik** [reply]
Not currently, no, but this should not be too difficult. Could you open up an issue describing the format it should be saved as? Is it miliseconds, number plus unit etc.

**Reply by ramytanios** [reply]
Will do 👍

**Reply by ramytanios** [reply]
https://github.com/VirtusLab/scala-yaml/issues/275
